src/content/docs/security.md

```diff
@@ -286,20 +290,31 @@ endpoint.
 We have dedicated machines for access directly to our production environment,
 access to such is restricted to a small number of engineers and is logged.
 
+### Intrusion Detection / Prevention / Monitoring
+
+We rely on Google Cloud network security for network-level intrusion detection.
+We do log all actions within our systems and do regularly review these logs as
+well as maintain alerting which is delivered to our engineering teams, both for
+our web products as well as our backend database and servers.
+
 ## Incident Response
 
 We have a private playbook for incident response. We have logging and alerting
 in place to detect and respond to incidents. We have both dedicated private
 channels on Discord for response as well as back-up text communication pathways
 as well as phone numbers for all engineers.
 
+We do not currently have separate incident tracking beyond our public GitHub. We
```

Comment:
Folks can also follow our status page (which you linked at the end) not sure if that’s worth mentioning here 🤷